### ISSUE TITLE
Toolshelf not working

### ISSUE THREAD

**Issue by lwalzer** [alessandro-zomparelli/tissue#12]
Hello! I tried to install Tissue Addon for Blender, but instead of the separate toolshelf I see in the tutorials I only get this small sub-shelf without the Tessellate options. What is wrong here? I use Blender 2.78.5

![grafik](https://user-images.githubusercontent.com/30218637/28251062-b42beb02-6a75-11e7-85bb-5dc113cd7027.png) My screen

![grafik](https://user-images.githubusercontent.com/30218637/28251065-c8722040-6a75-11e7-96de-6a9b37482a6c.png) How it should be (image by Blenderwiki)

Thanks in advance!

**Comment by warnotte** [comment]
Same for me with 2.78c

**Comment by warnotte** [comment]
Ok,

On the other issue, the answer is by the author : 

This issue should be fixed in the version 0.3.
In the new version there is no more a Tissue panel, but the operations are now in the "Create" panel and "Tools" panel.

**Comment by lwalzer** [comment]
Oh yes, you're right, thank you very much, I had not seen that, sorry! :-)

**Comment by diramazioni** [comment]
I could not find it till I lend up here. Maybe the docs should be updated or a description in the addon installation/user-settings highlighting the addon is splitt in two tabs

**Comment by alessandro-zomparelli** [comment]
Yes you are right, it's my bad. I have to update the documentation... 
Actually I'm not completely satisfied by this splitted panel. Maybe a single one in Tools is better? What do you think?
